### DISCUSSION TITLE
Issue Updating Raspberry Pi Image

### DISCUSSION THREAD

**Discussion by motion4life99** [dani-garcia/vaultwarden#1298] [Q&A]
Hi There!
I have a raspberry pi 3 B Arm7 that I have running bitwarden_rs with docker.

All has been well, but the past few months I've not been able to update it.  It says that the image is current, but it's been 5 months.  And now I'm starting to experience issues with the iphone app so I thought I would try updating the server first again and noticed that I can't.

command I'm running to update:  sudo docker pull bitwardenrs/server:raspberry
Then I get a hash and it says that the image is up to date.
So when I try to search for the image is says it was last updated 5 months ago

Any help would be greatly appreciated.  Thanks!

**Comment by BlackDex** [comment]
That tag is not working/updated anymore.

You should use either `latest`, `alpine` or `testing`, `testing-alpine` to get the latest version.
`latest` or `alpine` will match the latest tagged release (currently v1.18.0) and `testing` or `testing-alpine` is based upon the master branch and will be updated during every merge/push.

**Comment by motion4life99** [comment]
Got it.  Thanks for the quick response.  I will make sure to use latest going forward for my tag to pull.
